src/stories/Pages/MapOverview.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
```

Comment:
To load the CSS for leaflet (including references to icons), make sure to import it in the same file you are using the map.

```suggestion
import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
import 'leaflet/dist/leaflet.css';
```